ckanext/canada/plugins.py

```diff
@@ -37,6 +44,54 @@
 log = logging.getLogger(__name__)
 
 
+class CanadaDatasetsPlugin(SchemingDatasetsPlugin):
+    """
+    Plugin for dataset and resource
+    """
+    p.implements(p.IDatasetForm, inherit=True)
+
+    #IDatasetForm
+    def is_fallback(self):
+        # type: () -> bool
+        return False
+
+
+    #IDatasetForm
+    def package_types(self):
+        # type: () -> list[str]
+        return ['dataset','info','prop']
```

Comment:
Do we need to implement `is_fallback` and `package_types`? won't the parent implementation work?

Reply:
@wardi Yeah my bad, I had those in before I added the inherit on the implementation of DatasetForm.